src/clicache/src/CacheFactory.cpp

```diff
@@ -84,6 +85,12 @@ namespace Apache
 					//TODO::split
           SafeConvertClassGeneric::SetAppDomainEnabled(appDomainEnable);
 
+          if (appDomainEnable)
+          {
+            // Register managed AppDomain context with unmanaged.
+            apache::geode::client::createAppDomainContext = &Apache::Geode::Client::createAppDomainContext;
```

Comment:
I think we may want to just abandon the `appdomain-enabled` property and always enable AppDomain support when using the .NET client. I don't know that there is any good reason for running disabled since .NET uses AppDomains heavily (prior to .NET Core).